Sun_Light_Project copy.py

```diff
@@ -0,0 +1,28 @@
+from matplotlib import pyplot, dates
+from csv import reader
+from dateutil import parser
+
+with open('microbit-data-13-2022-09-54-23-0400.csv', 'r') as f:
+	data = list(reader(f, delimiter = "\t"))
+
+light = [float(i[1]) for i in data[2:-1]]
+time = [float(i[0]) for i in data[2:-1]]
```

Comment:
If you want to conver the time into hours. Time is a list. 

```
hour = []
for t in time:
  hour.append(t/60/60)
```
At the end, you can plot sun light vs. hours instead of sunlight vs. time(seconds)